### ISSUE TITLE
Crashes directly at first use after installation and initial setup

### ISSUE THREAD

**Issue by s99h** [florisboard/florisboard#809]
#### Environment information
- FlorisBoard Version: 0.3.11-beta04 (39)
- Device: HUAWEI MHA-L29
- Android: 7.0 (cn=N sdk=24)

#### Features enabled
```kt
smartbar = true
suggestions = false
suggestions_clipboard = true
suggestions_next_word = true
glide = false
clipboard_internal = true
clipboard_history = true
```

#### Attached stacktrace files
<details>
<summary>1620156267128.stacktrace</summary>

```
android.view.InflateException: Binary XML file line #2: Binary XML file line #2: Error inflating class <unknown>
Caused by: android.view.InflateException: Binary XML file line #2: Error inflating class <unknown>
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.newInstance0(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:430)
	at android.view.LayoutInflater.createView(LayoutInflater.java:656)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:798)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:738)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:495)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:426)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:377)
	at android.view.View.inflate(View.java:21031)
	at dev.patrickgold.florisboard.ime.media.emoji.EmojiKeyboardView.<init>(EmojiKeyboardView.kt:81)
	at dev.patrickgold.florisboard.ime.media.emoji.EmojiKeyboardView.<init>(EmojiKeyboardView.kt:63)
	at dev.patrickgold.florisboard.ime.media.emoji.EmojiKeyboardView.<init>(EmojiKeyboardView.kt:62)
	at dev.patrickgold.florisboard.ime.media.MediaInputManager.createTabViewFor(MediaInputManager.kt:164)
	at dev.patrickgold.florisboard.ime.media.MediaInputManager.access$createTabViewFor(MediaInputManager.kt:48)
	at dev.patrickgold.florisboard.ime.media.MediaInputManager$onRegisterInputView$1$4.invokeSuspend(MediaInputManager.kt:113)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at android.os.Handler.handleCallback(Handler.java:761)
	at android.os.Handler.dispatchMessage(Handler.java:98)
	at android.os.Looper.loop(Looper.java:156)
	at android.app.ActivityThread.main(ActivityThread.java:6524)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:941)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:831)
Caused by: java.lang.IllegalArgumentException: The style on this component requires your app theme to be Theme.AppCompat (or a descendant).
	at com.google.android.material.internal.ThemeEnforcement.checkTheme(ThemeEnforcement.java:243)
	at com.google.android.material.internal.ThemeEnforcement.checkAppCompatTheme(ThemeEnforcement.java:213)
	at com.google.android.material.internal.ThemeEnforcement.checkCompatibleTheme(ThemeEnforcement.java:148)
	at com.google.android.material.internal.ThemeEnforcement.obtainStyledAttributes(ThemeEnforcement.java:76)
	at com.google.android.material.tabs.TabLayout.<init>(TabLayout.java:508)
	at com.google.android.material.tabs.TabLayout.<init>(TabLayout.java:488)
	... 24 more

```
</details>

**Comment by patrickgold** [comment]
Thanks for your bug report! This is the second time this crash occurs on a Huawei phone in the last few days, strange. I am currently unsure if this bug always existed and was just undiscovered or if I introduced it recently. If possible, could you try to install the stable version 8v0.3.10) to see if the crash occurs there as well? This would massively help me in narrowing down where exactly I introduced this crash and what exactly I have to search for.

**Comment by s99h** [comment]
To narrow down the search I have just installed all versions beginning with 0.3.10.
0.3.10 and 0.3.11-beta01 are starting and running fine in this context - no crash.
The crash occurs with 0.3.11-beta02 and all later versions including 0.3.11-beta04.

If you need further input, please let me know. I'd be glad to help.

**Comment by patrickgold** [comment]
Above PR is a fix for the AppCompat theme crash, tested out on my physical Huawei P9 lite (crash was reproducible on that device). The fix will be included in v0.3.11-beta05!
